docs/README-Extending.md

```diff
@@ -103,7 +103,7 @@ Currently provided operations:
 - `initForZowe(type: string, meta: ICommandProfileTypeConfiguration[]): Promise<void>;`
 
 ```typescript
-const zoweExplorerApi = ZoweVsCodeExtension.getZoweExplorerApi("1.15.0");
+const zoweExplorerApi = ZoweVsCodeExtension.getZoweExplorerApi("1.16.1");
```

Comment:
@lauren-li Does this mean that we have to update this const everytime we do a release? or Maybe this is hard-coded for now and it will be changed in the future?

Reply:
@jellypuno This is just an example. Extenders can specify whichever version of the API they wish to target. I only updated it here because the examples in this document previously had inconsistent version numbers, which could be confusing for readers.